config/config.php

```diff
@@ -2,203 +2,285 @@
 
 declare(strict_types=1);
 
-use example\Type\ExampleType;
-use example\Query\ExampleQuery;
-use example\Mutation\ExampleMutation;
-use example\Type\ExampleRelationType;
+//use App\GraphQL\Types\ExampleType;
+//use App\GraphQL\Queries\ExampleQuery;
+//use App\GraphQL\Mutations\ExampleMutation;
+//use App\GraphQL\Types\ExampleRelationType;
 
 return [
 
-    // The prefix for routes
+    /*
+    |--------------------------------------------------------------------------
+    | The prefix for routes
+    |--------------------------------------------------------------------------
+    |
+    */
+
     'prefix' => 'graphql',
 
-    // The routes to make GraphQL request. Either a string that will apply
-    // to both query and mutation or an array containing the key 'query' and/or
-    // 'mutation' with the according Route
-    //
-    // Example:
-    //
-    // Same route for both query and mutation
-    //
-    // 'routes' => 'path/to/query/{graphql_schema?}',
-    //
-    // or define each route
-    //
-    // 'routes' => [
-    //     'query' => 'query/{graphql_schema?}',
-    //     'mutation' => 'mutation/{graphql_schema?}',
-    // ]
-    //
+    /*
+    |--------------------------------------------------------------------------
+    | The routes to make GraphQL request. Either a string that will apply
+    | to both query and mutation or an array containing the key 'query'
+    | and/or 'mutation' with the according Route
+    |--------------------------------------------------------------------------
+    |
+    | Example:
+    |
+    | Same route for both query and mutation
+    | 'routes' => 'path/to/query/{graphql_schema?}',
+    |
+    | or define each route
+    | 'routes' => [
```

Comment:
I'm a bit torn here, in particular for some practical reason: if a comment line starts with `//`, any editor/IDE I know allows you to define shortcuts to add/remove them in a whim, without having to play chase-the-cat and manually remove stuff (for example in PhpStorm it's <kbd>cmd </kbd>-<kbd>/</kbd>).

I would prefer "code samples" (not documentation) to remain with `//` and thus preferable outside of the regular `|` blocks.